draft-irtf-cfrg-frost.md

```diff
@@ -1128,6 +1127,29 @@ If `vss_verify` fails, the participant MUST abort the protocol, and failure shou
       return 0
 ~~~
 
+We now define how the coordinator and signing participants can derive group info, which is an input into the FROST signing protocol.
+
+~~~
+    derive_group_info(n, t, vss_commitment):
+
+    Inputs:
+    - n, the number of shares to generate, an integer
+    - t, the threshold of the secret sharing scheme, an integer
+    - vss_commitment: A VSS commitment to a secret polynomial f.
+
+    Outputs:
+    - PK, the public key representing the group
+    - signer_public_keys, a set of n public keys PK_i for i=1,...,n, where PK_i is the public key for participant i.
+
+    derive_group_info(n, t, vss_commitment)
+      PK = vss_commitment[0]
+      signer_public_keys = {}
+      for i in range(1, n):
+        PK_i = SUM(commitment[0], commitment[t-1]){A_j}: A_j*(i^j)
```

Comment:
We don't use this notation anymore -- can you please rewrite it using a loop?

Reply:
Addressed this and the loop in verify share.